api/scripts/data-generation/add-many-fake-members-related-to-one-organization.js

```diff
@@ -0,0 +1,90 @@
+const { knex } = require('../../lib/infrastructure/bookshelf');
+const DomainTransaction = require('../../lib/infrastructure/DomainTransaction');
+const { MembershipUpdateError, UserCantBeCreatedError, ForbiddenAccess } = require('../../lib/domain/errors');
+const _ = require('lodash');
```

Comment:
chipotage : 
Plutôt que d'appeler tout lodash, si on à besoin que du times, je propose
```suggestion
const times = require('lodash/times');
```

Reply:
Top ! je fais ça